### PR TITLE
React components improvements

### PR DESCRIPTION
**Pull request by MonkeyDo**
# Summary

* This is a…
    * (x) Bug fix
    * (x) Minor / simple change (like a typo)

# Problem

• Fixes recently introduced issue where user names would not appear in follow and recent pages
• Support older browsers, otherwise they would see an empty page. > 91% coverage now
• Some minor style adjustments

# Actions

Consider reinstating the jinja template table of listens (or showing a big ol` warning message) for unsupported browsers and disabled javascript users, instead of showing an empty page with no indication.

### CODE REVIEW

File: listenbrainz/webserver/static/js/jsx/follow-users.jsx [hunk 1]
```diff
@@ -75,15 +75,15 @@ export class FollowUsers extends React.Component {
               <tr>
                 <th colSpan="2" width="50px">Order</th>
                 <th>User</th>
-                <th width="50%">Listening now</th>
-                <th width="25px"></th>
-                <th width="85px"></th>
+                <th>Listening now</th>
```
Comment (mayhem) [inline listenbrainz/webserver/static/js/jsx/follow-users.jsx]: Beat me to reporting it!

Comment (MonkeyDo) [reply listenbrainz/webserver/static/js/jsx/follow-users.jsx]: I find this table quite ugly. Some redesign job in the future :)

**Comment by MonkeyDo** [comment]
> I really need to learn react and learn this code as well. Got tips on where to jump in?

Maybe an IRL Q&A session between to two of us?

Other than that I don't have any specific resource, but I can scan through a few basics tutorials and see what's most helpful.

File: listenbrainz/webserver/static/js/jsx/follow-users.jsx (hunk 1 above)
Comment (mayhem) [reply listenbrainz/webserver/static/js/jsx/follow-users.jsx]: Yes, please!

**Comment by mayhem** [comment]
I'd like that, but I should have a basic understanding first. So, yes please to tutorials.

**Pull request merged**